.github/workflows/OracleRunParallel.yml

```diff
@@ -25,9 +25,20 @@ jobs:
                sudo rm -rf "/usr/local/share/boost"
                sudo rm -rf "$AGENT_TOOLSDIRECTORY" 
 
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-          java-version: '1.8.0'
+          java-version: '8'
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Configure Liquibase
+        env:
+          GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
+        run: |
+          mvn -B versions:use-dep-version -Dincludes=org.liquibase:liquibase-core -DdepVersion=0-SNAPSHOT -DforceVersion=true
```

Comment:
Just synced with @nvoxland -- he is also going to return the AWS & Oracle workflows to what they were before, i.e running against the last released version of Liquibase, and not in flight snapshot versions.

Reply:
I took the changes in these files out completely.